ci/Jenkinsfile.tests

```diff
@@ -64,6 +64,10 @@ pipeline {
               set -eo pipefail
               make test 2>&1 | tee -a ${LOG_FILE}
             """
+            sh """#!/bin/bash
+              set -eo pipefail
+              make component-test 2>&1 | tee -a ${LOG_FILE}
```

Comment:
This should probably be it's own stage. Can they run in parallel?

Reply:
yeah, imo this would be better to have as it's own job. The two tests types are completely separate to each other so running in parallel should be no issue.

Reply:
Separate job is overkill, I mean just using parallel functionality in a single Jenkinsfile.

Reply:
Sounds good 👍